src/promptflow/promptflow/_proxy/_proxy_factory.py

```diff
@@ -5,14 +5,14 @@
 from typing import Dict, Type
 
 from promptflow._constants import FlowLanguage
-from promptflow._proxy import AbstractInspectorProxy
-from promptflow._utils.async_utils import async_run_allowing_running_loop
+from promptflow._proxy._base_proxy_factory import BaseProxyFactory
 from promptflow.batch._base_executor_proxy import AbstractExecutorProxy
 from promptflow.batch._csharp_executor_proxy import CSharpExecutorProxy
 from promptflow.batch._python_executor_proxy import PythonExecutorProxy
+from promptflow.batch._chat_group_orchestrator_proxy import ChatGroupOrchestratorProxy
 
 
-class ProxyFactory:
+class ProxyFactory(BaseProxyFactory):
```

Comment:
why we need multiple factories?

Reply:
Per offline synced, use local import as a walk around.